server/src/main/java/com/linecorp/centraldogma/server/storage/StorageManager.java

```diff
@@ -95,6 +95,18 @@ default T create(String name, Author author) {
      */
     T unremove(String name);
 
+    /**
+     * Purges a set of names for the elements which have been marked for purging.
+     */
+    void purgeMarked();
+
+    /**
+     * Marks an element with the specified {@code name} for purge.
```

Comment:
Marks the specified {@code name} element for purging?

Reply:
We were supposed to use `markForPurge`. https://github.com/line/centraldogma/pull/416#discussion_r303713795
I'm going to change `element for purge`. :-)